script.js

```diff
@@ -1,4 +1,308 @@
+var playerOneRoll = [];
+var playerTwoRoll = [];
+var playerOneScore = 0;
+var playerTwoScore = 0;
+var playerOneChoice;
+var playerTwoChoice;
+var gameModes = [
+  "normal-highest",
+  "normal-lowest",
+  "auto-highest",
+  "auto-lowest",
+];
+var currentGameMode = "game start";
+var currentPlayer = "Player One";
+var numDice = 2; //number of dice defaults at 2 in case player did not choose any, will change on game modes
+var winner = "";
+//assign who is playing
+var playerOneIsPlaying = true;
+var playerTwoIsPlaying = false;
+var compareScores = false;
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (currentGameMode == "game start") {
+    currentGameMode = "choose number of dice";
+    console.log(currentGameMode);
+    return `How many dice you want to play?`;
+  }
+
+  if (currentGameMode == "choose number of dice") {
```

Comment:
nice work! i think you could separate all different game logics into their own functions and call them each time a currentGameMode is checked. Makes your code neater and easier to reads when you come back to it after a while.